src/api/from/from_array.rs

```diff
@@ -61,6 +61,7 @@ macro_rules! impl_from_array {
                 mod [<$id _from>] {
                     use super::*;
                     #[test]
+                    #[cfg_attr(miri, ignore)]
```

Comment:
This test itself does something weird and I don't have a good idea what the intention here was. I'm open to re-enabling it, the benefit of ignoring it is that `cargo miri test -- array` fully passes.